src/main/java/lotto/domain/LastWeekWinnerNumber.java

```diff
@@ -0,0 +1,23 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class LastWeekWinnerNumber {
+	private final LottoGame winnerNumber;
+
+	public LastWeekWinnerNumber(String inputNumber) {
+		String[] splitNumbers = inputNumber.split(",");
+		winnerNumber = new LottoGame(() -> {
+			List<LottoNumber> gameNumberList = new ArrayList<>();
+			Arrays.stream(splitNumbers)
+				.forEach((numbers) -> gameNumberList.add(new LottoNumber(Integer.valueOf(numbers.trim()))));
+			return gameNumberList;
+		});
```

Comment:
해당 생성자에서 LottoGame을 생성하는 로직을 수행하는 것보다
LottoGame에게 inputNumber를 넘겨주어 생성하도록 해준다면 어떤 장점이 있을까요 :) ?

```suggestion
	public LastWeekWinnerNumber(String inputNumber) {
		winnerNumber = new LottoGame(inputNumber);
```